wrappers/android/zxingcpp/build.gradle.kts

```diff
@@ -116,6 +121,10 @@ publishing {
 }
 
 signing {
+    setRequired {
```

Comment:
Happy to remove this or change the implementation. I added it because it allows us to run `./gradlew publishToMavenLocal` and use the `mavenLocal()` repository rather than manually including the aar for testing.

I borrowed the implementation from this [discussion on the gradle forums](https://discuss.gradle.org/t/how-to-only-enable-the-signing-plugin-if-were-about-to-release/39555)